File: php/src/app/dominio/adapters/AdapterConcretoA.php
<?php
namespace dominio\adapters;


class AdapterConcretoA implements IAdapter {
    /**
     * @var AdaptadaA
     */
    private $adaptada;

    public function setAdaptada(AdaptadaA $adaptada){
        $this->adaptada = $adaptada;
    }

    public function operacion(){
        $this->adaptada->operacionRealA();
    }
}

File: java/src/main/java/dominio/adapters/AdapterConcretoA.java
package dominio.adapters;

public class AdapterConcretoA implements IAdapter{
    private AdaptadaA adaptada;

    public void setAdaptada(AdaptadaA adaptada) {
        this.adaptada = adaptada;
    }

    @Override
    public void operacion() {
        this.adaptada.operacionRealA();
    }
}